sql_user_check_looker/run.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+
+set -e
+
+readonly URL=$URL
+readonly CLIENT_ID=$CLIENT_ID
+readonly CLIENT_SECRET=$CLIENT_SECRET
+
+response=$(curl -s -d "client_id=$CLIENT_ID&client_secret=$CLIENT_SECRET" "$URL/api/4.0/login")
```

Comment:
Check if status code was 200. Raise error otherwise. Same comment for other occurences

Reply:
done.